newIDE/app/src/ObjectsList/index.js

```diff
@@ -295,15 +295,12 @@ export default class ObjectsList extends React.Component<Props, State> {
 
   _rename = (objectWithContext: ObjectWithContext, newName: string) => {
     const { object } = objectWithContext;
-
-    if (getObjectWithContextName(objectWithContext) === newName) {
-      return;
-    }
-
     this.setState({
       renamedObjectWithContext: null,
     });
 
+    if (getObjectWithContextName(objectWithContext) === newName) return;
```

Comment:
Make sure to merge again master in your branch to avoid this file from appearing

Reply:
(if this does not solve the problem, you can either ignore this or close this PR and open another - I think it's a GitHub bug)